fly/commands/workers.go

```diff
@@ -191,3 +194,29 @@ func (w *worker) VersionCell() ui.TableCell {
 
 	return column
 }
+
+func (w *worker) ageCell() ui.TableCell {
+	var column ui.TableCell
+
+	const minute = 60
+	const hour = minute * 60
+	const day = hour * 24
+
+	age := time.Now().Unix() - w.StartTime
+	if w.StartTime <= 0 || age < 0 {
+		column.Contents = "n/a"
+		column.Color = color.New(color.Faint)
+	} else {
+		if age/day > 0 {
+			column.Contents = fmt.Sprintf("%dd", age/day)
+		} else if age/hour > 0 {
+			column.Contents = fmt.Sprintf("%dh%dm", age/hour, (age%hour)/minute)
+		} else if age/minute > 0 {
+			column.Contents = fmt.Sprintf("%dh%dm", age/minute, age%minute)
```

Comment:
whoops. turns out deleting tests masked the bug!
```suggestion
			column.Contents = fmt.Sprintf("%dm%ds", age/minute, age%minute)
```

Reply:
@evanchaoli @analytically is correct

Reply:
#4545 tracks this

Reply:
Oh, that's a stupid error. Let's me raise a PR to fix that.